.github/workflows/releases.yml

```diff
@@ -93,5 +93,73 @@ jobs:
     - name: Wait for Metabase to start
       run: while ! curl -s localhost:3000/api/health; do sleep 1; done
       timeout-minutes: 3
+    - name: Check API health
+      run: curl -s localhost:3000/api/health
+
+  verify-container-ee:
+    runs-on: ubuntu-20.04
+    needs: release-ee
+    timeout-minutes: 10
+    steps:
+    - uses: actions/download-artifact@v2
+      name: Retrieve previously prepared JAR
+      with:
+        name: metabase-jar-ee
+        path: github-release
+    - name: Show its version.properties
+      run: jar xf github-release/metabase.jar version.properties && cat version.properties
+    - name: Get the tag from version.properties
+      id: version
+      run: echo "::set-output name=tag::$(grep tag version.properties | cut -d = -f 2)"
```

Comment:
Pardon the pragmatic choice of `cut` here, instead of a fancy regexp/awk/Perl (guaranteed to be longer) since I'm hesitated to come up with something which can take into account current (e.g. `0.40.1-rc2`) and future (who knows? `0.44.2.3.3-beta2`) format for the tag.